src/mds/MDCache.h

```diff
@@ -215,6 +215,18 @@ class MDCache {
 
   void advance_stray();
 
+  bool get_export_ephemeral_distributed_config(void) const {
+    return export_ephemeral_distributed_config;
+  }
+
+  bool get_export_ephemeral_random_config(void) const {
+    return export_ephemeral_random_config;
+  }
+
+  void ephemeral_pin(elist<CInode*>::item& in) {
+    ephemeral_pins.push_back(&in);
+  }
```

Comment:
I'm confused how this change reappeared. It's causing build failures.

Reply:
I think I messed it up when I rebased after that merge conflict. I'm cleaning everything up now